src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(...args) {
+    if (args.length === 0) {
```

Comment:
you can use single variable here, this function expects only one argument

```suggestion
  function adder(num) {
    if (!num) {
```